pmlc/rt/opencl/BUILD

```diff
@@ -9,4 +9,9 @@ cc_test(
     srcs = ["clprobe.cc"],
     deps = ["@opencl_icd_loader", "@opencl_hpp_headers"],
     size = "small",
+    tags = [
+      "manual",
+      "skip_macos",
+      "skip_windows",
```

Comment:
How are CI builds passing without these tags?

Reply:
Maybe the MacOS machines don't have the latest Xcode installed.

Reply:
And windows?

Reply:
Well, to me it looked it was not tested on Windows. The changes now look differently... No skipping.